.github/workflows/test.yml

```diff
@@ -141,10 +141,12 @@ jobs:
           ${{ github.job }}-${{ runner.os }}-go-2-${{ hashFiles('**/go.sum') }}
           ${{ github.job }}-${{ runner.os }}-go-2-
     - name: test
+      run: go run ./cmd/testwrapper ./...
+    - name: bench all
       # Don't use -bench=. -benchtime=1x.
       # Somewhere in the layers (powershell?)
       # the equals signs cause great confusion.
-      run: go run ./cmd/testwrapper ./... -bench . -benchtime 1x
+      run: ./tool/go test ./... -bench . -benchtime 1x -run=^$ 
```

Comment:
Per the comment above, the `-run=^$` may not work, we might need `-run '^$'` instead

Reply:
oops, good catch!